windows_docker_resources/Dockerfile.dashing

```diff
@@ -58,7 +58,10 @@ RUN C:\TEMP\python-37.exe /quiet `
 RUN powershell -noexit "Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))"
 
 # choco installs
-RUN choco install -y cmake curl git vcredist2013 vcredist140 cppcheck patch
+RUN choco install -y cmake curl git vcredist2013 vcredist140 patch
+# Pin cppcheck until our builds are compatible with 2.0 https://github.com/ros2/ros2/issues/942
+RUN choco install -y cppcheck --version 1.90
+RUN choco pin add --name=cppcheck --version 1.90
```

Comment:
For whatever reason last time I pinned with chocolatey neither `--name cppcheck` nor `--version=1.90` worked so this command uses two different argument forms.

Reply:
Does `choco pin` by itself just apply to all packages?

Reply:
> Does `choco pin` by itself just apply to all packages?

Nope. It lists existing pins. The --name argument is required for `choco pin add` https://chocolatey.org/docs/commands-pin